### DISCUSSION TITLE
How extend SelectField class?

### DISCUSSION THREAD

**Discussion by illambo** [rebing/graphql-laravel#958] [Q&A] [answered]
Hi! 
Is there any way to customize (extend) the SelectField class? 

I'm checking [this](https://github.com/rebing/graphql-laravel/blob/8567827b3589af7e8a1a872d8489f01249b122e6/src/Support/Field.php#L273) but from the configuration I don't think there is such a possibility.

Thanks.

**Comment by mfn** [comment, marked as answer]
> but from the configuration I don't think there is such a possibility.

No, but you can define it "per type" or query etc.

But basically yes, you discovered already the extent to which this is customizable.

**Reply by illambo** [reply]
Thanks, i didn't realize queries and mutations extended the Field class.

**Reply by mfn** [reply]
lol yeah, not obvious at first 😬
